src/vendors/explorer/apis/AccountsApi.ts

```diff
@@ -381,14 +381,10 @@ export class AccountsApi extends runtime.BaseAPI {
     /**
     */
     async getValidatorsList(requestParameters: GetValidatorsListRequest): Promise<Array<ValidatorRow>> {
-      try {
         const response = await this.getValidatorsListRaw(requestParameters);
         return await response.value();
-      } catch (e) {
-        // TODO: Properly handle this error - should show a warning or so! See #403
-        return [];
-      }
     }
+
```

Comment:
this came from reverting the hotfix commit. should we keep it deleted?

Reply:
If catching the error above works fine, then go ahead!